cmd/package_test.go

```diff
@@ -203,3 +203,69 @@ func (s *suite) TestDescribePackage(c *C) {
 	c.Assert(err, IsNil)
 	c.Check(descr, MatchesMultiline, fmt.Sprintf(".*PACKAGE DOCUMENTATION\n%s\n", DefaultText))
 }
+
+func (s *suite) TestRecursiveRunYamls(c *C) {
+	// Prepare.
+	s.importFakeOSvBootstrapPkg(c)
```

Comment:
I was wondering how can the run config be included since there was no `require` directive in the package yaml. Then I realised that `osv.bootstrap` is the package that is always (implicitly) included in any package that we compose. Is this correct?

I am just double-checking this as it was somewhat confusing to me with no `require` directive in the package yaml.